docker-compose.staging.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+# Production environment override
+services:
+  php:
+    image: 'mspserver-php-staging'
+    build:
+      context: .
+      target: php_prod
+    volumes:
+      - ./:/srv/app
+      # If you develop on Mac or Windows you can remove the vendor/ directory
+      #  from the bind-mount for better performance by enabling the next line:
+      - /srv/app/vendor
+      - /srv/app/var/cache
+    environment:
+      URL_WEB_SERVER_HOST: ${URL_WEB_SERVER_HOST:-localhost}
```

Comment:
default set to localhost so a .env.local wouldn't be required if we just want to quickly set up a local staging environment